src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http3.cs

```diff
@@ -421,7 +443,7 @@ public async Task Public_Interop_Upgrade_Success(string uri)
             {
                 Method = HttpMethod.Get,
                 RequestUri = new Uri(uri, UriKind.Absolute),
-                Version = HttpVersion.Version30,
+                Version = HttpVersion.Version20,
```

Comment:
Does this actually matter? The first request should not be processed via HTTP3 regardless, because we haven't received the Alt-Svc header yet. In fact isn't that the point of setting Version30 here -- to validate that even though we requested 30, we won't get it on the first request because we haven't gotten an Alt-Svc yet?